### PR TITLE
Code Clean Up

### PR DESCRIPTION
**Pull request by allanjoroge**
Created Interface for Registration controller 
Moved helper methods to util classes

### CODE REVIEW

**Review by fredthuo1 — APPROVED** [review]
Looks good!

File: pom.xml [hunk 1]
```diff
@@ -137,10 +137,6 @@
                 <artifactId>spring-boot-maven-plugin</artifactId>
                 <configuration>
                     <excludes>
-                        <exclude>
-                            <groupId>org.projectlombok</groupId>
```
Comment (fredthuo1) [inline pom.xml:141]: Does it throw any errors once you take this off?

File: src/main/java/com/example/demo/controller/RegistrationRestControllerImpl.java [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+        return new GenericResponse("success");
+    }
+
+    @GetMapping("user/activateAccount")
+    public GenericResponse activateAccount(@RequestParam final String email) {
+        final User registered = userService.findUserByEmail(email);
+        final String token = UUID.randomUUID().toString();
+        mailSender.send(registrationUtil.sendAccountActivationEmail(token, registered.getEmail(), registered));
+        return new GenericResponse("success");
+    }
```
Comment (fredthuo1) [inline src/main/java/com/example/demo/controller/RegistrationRestControllerImpl.java]: This is neat!!!!

File: pom.xml (hunk 1 above)
Comment (allanjoroge) [reply pom.xml:141]: no

**Pull request merged**